gamestonk_terminal/comparison_analysis/ca_controller.py

```diff
@@ -312,6 +316,10 @@ def call_technical(self, other_args: List[str]):
         """Process technical command"""
         finviz_compare_view.screener(other_args, "technical", self.ticker, self.similar)
 
+    def call_po(self, _):
+        """Open Portfolio Optimization menu with ticker and similar"""
+        port_optimization_controller.menu([self.ticker] + self.similar)
```

Comment:
I would do something more convolved here, like:

User does
Menu -> Comparison Analysis -> Portfolio Optimization

* when they `quit`, they quit the program.
* when the `q` the go to the `Menu` instead of back to the Comparison Analysis.

Reply:
Yeah I am not sure exactly how to do that.  If I do either `q` or `quit` from menu-ca-po, it just goes back to ca.

Reply:
You just need to do:
```
return port_optimization_controller.menu([self.ticker] + self.similar)
```

Because `port_optimization_controller.menu` already returns q/quit